tower_eval/tasks/generate.py

```diff
@@ -80,13 +86,42 @@ def generate(i: int, config_path: str, config_type: str) -> None:
 
                 model.source_language = src_lang
                 model.target_language = tgt_lang
-            model.generation_with_resume(input_file=input_file, output_file=output_file, metadata=configs, metadata_file=metadata_file)
-            add_average_generation_time(output_file, metadata_file, language=tgt_lang, mode=average_time_metric)
-    try:
-        model.server.close_server()
-    # in case model does not have a close server attribute (e.g., openAI)
-    except AttributeError:
-        pass
+            model.generation_with_resume(
+                input_file=input_file,
+                output_file=output_file,
+                metadata=configs,
+                metadata_file=metadata_file,
+            )
+            add_average_generation_time(
+                output_file, metadata_file, language=tgt_lang, mode=average_time_metric
+            )
+
+
+def simple_generate(
+    input_paths: list[str],
+    output_paths: list[str],
+    model_path: str,
+    model_type: str,
+    model_args: dict,
+    metadata_file_paths: list[str],
+):
+    model_path_key = "model_dir" if model_type == "vllm" else "model"
+    model_args[model_path_key] = model_path
+    print(model_args)
```

Comment:
Can you please use log instead of print?

Reply:
accidentally left this here